src/gt4py/analysis/passes.py

```diff
@@ -821,18 +895,18 @@ def _make_stage(self, ij_block):
 
         return stage
 
-    def _make_apply_block(self, interval_block):
-        # Body
-        stmts = []
-        for stmt_info in interval_block.stmts:
-            if not isinstance(stmt_info.stmt, gt_ir.Decl):
-                stmts.append(stmt_info.stmt)
-        body = gt_ir.BlockStmt(stmts=stmts)
-        result = gt_ir.ApplyBlock(
-            interval=self._make_axis_interval(interval_block.interval), body=body
-        )
-
-        return result
+    # def _make_apply_block(self, interval_block):
+    #     # Body
+    #     stmts = []
+    #     for stmt_info in interval_block.stmts:
+    #         if not isinstance(stmt_info.stmt, gt_ir.Decl):
+    #             stmts.append(stmt_info.stmt)
+    #     body = gt_ir.BlockStmt(stmts=stmts)
+    #     result = gt_ir.ApplyBlock(
+    #         interval=self._make_axis_interval(interval_block.interval), body=body
+    #     )
+
+    #     return result
```

Comment:
Should this be deleted?

Reply:
I uncommented the code (reset it back to unchanged) in this PR. Could be deleted in a follow up.